nat/traversal/pinger.go

```diff
@@ -330,18 +305,33 @@ func (p *Pinger) multiPing(ip string, localPorts, remotePorts []int, initialTTL
 }
 
 func (p *Pinger) singlePing(remoteIP string, localPort, remotePort, ttl int, stop <-chan struct{}) (*net.UDPConn, error) {
-	conn, err := p.getConnection(remoteIP, remotePort, localPort)
+	conn, err := net.ListenUDP("udp4", &net.UDPAddr{Port: localPort})
```

Comment:
This is smart. :) Did you have a change to test with those mobile clients this code already?

Reply:
It works with our mobile client, just tested it. The provider with a cellular internet is not tested yet.